Header header

sensor_msgs/Image image
geometry_msgs/PoseStamped pose